pkg/apis/forklift/v1beta1/BUILD.bazel

```diff
@@ -30,6 +30,6 @@ go_library(
         "//vendor/k8s.io/client-go/rest",
         "//vendor/sigs.k8s.io/controller-runtime/pkg/client",
         "//vendor/sigs.k8s.io/controller-runtime/pkg/client/config",
-        "//vendor/sigs.k8s.io/controller-runtime/pkg/runtime/scheme",
+        "//vendor/sigs.k8s.io/controller-runtime/pkg/scheme",
```

Comment:
it can simplify the review if we extract the change of the dependencies to a separate PR

Reply:
yeah, given the amount of code changes needed along with the version bumps it's probably better done separately